packages/safety-suite-api/src/api/eForms.ts

```diff
@@ -184,3 +184,15 @@ export function getSignatureSignUrl(
     apiOptions
   });
 }
+
+export function getDriverByEmployeeId(
+  employeeId: string,
+  apiOptions: ApiOptions = {}
+): Request<User> {
```

Comment:
Don't we need to wrap it in an ApiResponse? I'm guessing we will need to update the backend for this.

Reply:
Login is still has no "response" wrapper, so this endpoint is just follow login response logic.